src/Root.cpp

```diff
@@ -159,6 +159,14 @@ void cRoot::Start(std::unique_ptr<cSettingsRepositoryInterface> a_OverridesRepo)
 	m_MojangAPI = new cMojangAPI;
 	bool ShouldAuthenticate = settingsRepo->GetValueSetB("Authentication", "Authenticate", true);
 	m_MojangAPI->Start(*settingsRepo, ShouldAuthenticate);  // Mojang API needs to be started before plugins, so that plugins may use it for DB upgrades on server init
+	AString MaxLoadedChunks = settingsRepo->GetValueSet("Server", "MaxChunkRAM", "30");
+	m_MinimizeRam = settingsRepo->GetValueSetB("Server", "MinimizeRAM", true);
```

Comment:
How about using a string value, "MemoryStrategy", instead? This way we could possibly add new strategies later on and it would be easier to understand.